autogpt_platform/frontend/src/components/tutorial.ts

```diff
@@ -1,6 +1,6 @@
-import { sendGAEvent } from "@next/third-parties/google";
 import Shepherd from "shepherd.js";
 import "shepherd.js/dist/css/shepherd.css";
+import { sendGAEvent } from "./analytics/google-analytics";
```

Comment:
```suggestion
import { sendGAEvent } from "@/components/analytics/google-analytics";
```

Reply:
We are keeping the same API so it can be copy pasted